build-docker-image.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
Super minor point, but since the .sh and .bat are the same content, and there's nothing shell-specific in there, we could just have one file for both.

Reply:
This came from HAPI and I'm not super familiar with windows - can I just delete one of them?

Reply:
Deleted the .sh and added execute permissions to the .bat version